core/templates/dev/head/domain/exploration/StatesObjectFactorySpec.js

```diff
@@ -203,7 +239,10 @@ describe('States object factory', function() {
             customization_args: {},
             default_outcome: {
               dest: 'new state',
-              feedback: [],
+              feedback: {
+                contnet_id: 'default_outcome',
```

Comment:
Here and in about 20 other places, content*

Reply:
Done!